src/lib/build.ts

```diff
@@ -44,7 +47,12 @@ export default async function run({
   loadMode = 'document',
   sandbox = true,
 }: SaveOption) {
-  const stat = fs.statSync(input);
+  const stat = await statPromise(input).catch((err) => {
```

Comment:
Could you apply the same improvement to lib/preview.ts ?

Reply:
Sure thing :)